apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,196 @@ const data: FoundAnimalFlow = {
+          "- It is cold, wet, or crying nonstop.",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the squirrel"),
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Does the squirrel have a fluffed-out tail, a body longer than 6 inches (excluding the tail), or is approaching humans/pets?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "This is likely a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: ["Is the squirrel alone?"],
+                    options: [
+                      {
+                        name: "Yes",
+                        flow: {
+                          prompt: [
+                            "You've likely found an infant squirrel, who needs to be guided back to their mother.",
+                            "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds. Then, wrap the sock in a soft towel and place it with the baby squirrel in an open container (e.g. a box) reheating the rice every two hours.",
+                            "Do not attempt to feed or give water to the squirrel, as this may lead to it imprinting on humans and becoming dependent on them.",
+                            "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found). If you're unsure of the right tree, or if the nest was destroyed, choose a tree closest to where the squirrel was found.",
+                            "Observe the baby for the next 6-8 hours, reheating the rice every two hours or so to keep it warm.",
+                            "Has the mother returned?",
+                          ],
+                          options: [
+                            {
+                              name: "Yes",
+                              flow: {
+                                prompt: macros.general.leave("the squirrel"),
+                              },
+                            },
+                            {
+                              name: "No",
+                              flow: {
+                                prompt: macros.general.rehab(
+                                  "the squirrel",
+                                  "If you are sure the parents are not nearby, and do not return within a few hours",
+                                ),
+                              },
+                            },
+                          ],
```

Comment:
Instead of guiding on direct intervention here, it would be referred to just direct to the rehab macro immediately (with the same "If you are sure the parents are not nearby, and do not return within a few hours" context).